source/javascripts/controllers/_AndroidKeystoreController.js.erb

```diff
@@ -101,7 +110,7 @@ angular.module("BitriseWorkflowEditor").controller("AndroidKeystoreController",
 			viewModel.actionProgress.success();
 
 			viewModel.uploadedKeystoreFile = new KeystoreFile("");
-			loadKeystoreFile();
+			loadKeystoreFile(true);
```

Comment:
can we change it to resolve logic by using `shouldHideLoading` or sth like that, so we concentrate `true` parameter passing to recursion only.